src/openapi-generator.ts

```diff
@@ -632,6 +632,13 @@ export class OpenAPIGenerator {
       const regexCheck = this.getZodStringCheck(zodSchema, 'regex');
       return {
         type: 'string',
+        // FIXME: https://github.com/colinhacks/zod/commit/d78047e9f44596a96d637abb0ce209cd2732d88c
+        minLength: Number.isFinite(zodSchema.minLength)
+          ? zodSchema.minLength ?? undefined
```

Comment:
I don't think you need the nullish coalescing (`??`) here, do you? If it is `null` (new versions) - the condition above would evaluate to false and we would return `undefined` anyway.